topics/admin/faqs/ansible_define-once.md

```diff
@@ -6,4 +6,25 @@ layout: faq
 contributors: [natefoo, shiltemann]
 ---
 
-Using practices like those shown above helps to avoid problems caused when paths are defined differently in multiple places. The datatypes config file will be copied to the same path as Galaxy is configured to find it in, because that path is only defined in one place. Everything else is a reference to the original definition! If you ever need to update that definition, everything else will be updated accordingly.
+Using variables, either by defining them ahead of time, or simply accessing them via existing data structures that have been defined
```

Comment:
Looks like the sentence was truncated?

Reply:
that was the end, added an `e.g. :` to make clear it means the example that follows.